src/Settings.elm

```diff
@@ -0,0 +1,295 @@
+module Settings exposing (State, Msg, init, update, view)
+
+import Html exposing (Html)
+import Html.Attributes
+import Html.Events
+import Form exposing (Form)
+import Form.Field as Field
+import Form.Input as Input
+import Form.Validate as Validate exposing (..)
+import Form.Error as Error exposing (..)
```

Comment:
Q: I see that you are using `(..)` for forms but you use qualified imports for stuff like `Html`. Can you explain why you prefer to use `Form` this way?

Reply:
I typically expose specific functions from the `Html` and `Html.Attributes` modules to make it easier to write and to avoid name collisions. But I wanted to be consistent with the style you were already using in the `Main.elm` file, so I used fully qualified names for the `Html` stuff.

Reply:
My reasoning is to expose only types and have everything else qualified. This avoids collisions automatically, it makes Intelisense to be really helpful but the biggest benefit I see is the code readability. Because if I am using qualified imports for everything outside the module, I know that every function without module prefix is a local function and I can just find it in the file directly. When there are some functions imported, I have hard time to look whether it is something from this module or something different. And if it is not in the module, then which module it belongs to.

Also it makes it easier to add/remove functions while developing, because I don't need to jump up and down the file to add/remove function from expose.

But that is my approach to imports. I really appreciate that you kept the original style of those Html imports.